src/server/api/routes/questions.router.js

```diff
@@ -0,0 +1,200 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const questionsController = require('../controllers/questions.controller');
+
+/**
+ * @swagger
+ * /questions:
+ *  get:
+ *    summary: Get all questions
+ *    description:
+ *      Will return all questions.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
```

Comment:
I dont think swagger work without those comments...... we just used the constructor  that what we found in the modules expamle that made from Zaki in the Project .....

Reply:
@senner007 
This how it was done for Class 11 FP. 
The idea is to keep the documentation together with the endpoint instead of a separate Swagger.JSON file. It keeps the code and documentation organized and easier to maintain. 
Zaki decided to add this to the boilerplate going forward. 
It uses JS-Docs, Swagger, and comments are in YAML format to generate the documentation.